tool/tctl/common/plugin/awsic.go

```diff
@@ -71,6 +72,22 @@ func (a *awsICArgs) validate(ctx context.Context, log *slog.Logger) error {
 	return nil
 }
 
+func (a *awsICArgs) validateSystemCredentialInput() error {
+	if !a.useSystemCredentials {
+		return trace.BadParameter("--useSystemCredentials must be set")
```

Comment:
I think the flag name is incorrect?
```suggestion
		return trace.BadParameter("--use-system-credentials must be set")
```